exec/java-exec/src/main/resources/bootstrap-storage-plugins.json

```diff
@@ -6,11 +6,13 @@
       workspaces: {
         "root" : {
           location: "/",
-          writable: false
+          writable: false,
+          physicalLocation: "file:///"
```

Comment:
Looks like "physicalLocation" could be constructed as "connection" + "location" ?